test/parallel/parallel.status

```diff
@@ -26,8 +26,6 @@ test-async-context-frame: PASS, FLAKY
 test-runner-run-watch: PASS, FLAKY
 # https://github.com/nodejs/node/issues/56794
 test-fs-cp: PASS, FLAKY
-# https://github.com/nodejs/node/issues/59090
-test-inspector-network-fetch: PASS, FLAKY
```

Comment:
This is still failing in the CI: https://github.com/nodejs/reliability/blob/main/reports/2025-08-26.md